src/main.rs

```diff
@@ -263,3 +268,26 @@ fn shutdown() -> eyre::Result<impl std::future::Future<Output = ()>> {
         }
     }))
 }
+
+fn init_tracing() -> eyre::Result<()> {
+    let fmt = tracing_subscriber::fmt::layer().with_ansi(stdout().is_terminal());
+
+    let env = match std::env::var("RUST_LOG") {
```

Comment:
It seems little different from this https://github.com/edgehog-device-manager/edgehog-device-runtime/blob/release-0.9/src/main.rs. Why?

Reply:
I think this was copied from the e2e-test. Where two directives are used in the default, since you cannot pass 2 directives to `EnvFilter::builder().with_default_directive()`.

I would change this to use the one in the `edgehog-device-runtime` since it's clearer.